### ISSUE TITLE
deployment/run-unit-tests.sh is errorneous

### ISSUE THREAD

**Issue by htrappmann** [awslabs/aws-limit-monitor#15]
The first "cd ../source/services/limitreport" works but then the second "cd ../source/services/slacknotify" fails because it wants to change to: "source/services/source/services/slacknotify".

**Comment by htrappmann** [comment]
Yes, thats what I do, I run it from deployment folder.

**Comment by thomasphorton** [comment]
It looks like each of the paths is constructed relative to the deployment directory, rather than the current directory.

It should look like this:

```
cd ../source/services/limitreport
npm install
npm test

cd ../slacknotify
npm install
npm test

cd ../tarefresh
npm install
npm test

cd ../customhelper
npm install
npm test
```

@htrappmann if you replace it with this it should unblock you.

**Comment by htrappmann** [comment]
Yes, thank you, did already :)

**Comment by shsenior** [comment]
Addressed with v5.1
